src/app/public/modules/split-view/split-view-drawer.component.scss

```diff
@@ -0,0 +1,58 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
+
+.sky-split-view-drawer {
+  border-right: 1px solid $sky-border-color-neutral-medium;
+  overflow: auto;
+  height: 100%;
+  width: 100%;
+}
+
+.sky-split-view-resize-handle {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  height: 100%;
+  width: 14px;
+  position: absolute;
+  z-index: 999;
+  cursor: ew-resize;
+  padding: 0;
+  border: 0;
+  background: transparent;
+  display: block;
+  top: 0;
+  bottom: 0;
+
+  &::-moz-range-thumb, &::-moz-range-track {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    width: 0;
+    height: 0;
+    border-radius: 0px;
+    border: 0 none;
+    background: none;
+    display: none;
+  }
+
+  &::-ms-thumb {
```

Comment:
I think we need to have styles for the `ms-track` too. I see the track portion when serving in IE

Reply:
Done. Nice catch!